src/main/java/Duke.java

```diff
@@ -1,10 +1,165 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    //CONSTANTS
+    public static final String SEPARATOR = "____________________________________________________________\n";
+
+    //METHODS
+    public static void printGreeting() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello from\n"
+                + logo
+                + SEPARATOR
+                + " Hello! I'm Duke\n"
+                + " What can I do for you?\n"
+                + SEPARATOR);
+    }
+
+    public static void printError() {
+        System.out.println(SEPARATOR
+                + " Oops, something went wrong!\n"
+                + SEPARATOR);
+    }
+
+    public static String[] organize(String userInput) {
+        String[] organizedUserInput = new String[3];
+        if (!userInput.contains(" ")) {
+            //command is list
+            organizedUserInput[0] = userInput;
+        } else {
+            int spaceIndex = userInput.indexOf(" ");
+            String command = userInput.substring(0, spaceIndex);
+            organizedUserInput[0] = command;
```

Comment:
Good that you defined another variable, command. Made your code very easy to read!